### PR TITLE
AnomalyDetector: multiple segments labeling #658

### LINKED ISSUE #658: AnomalyDetector: multiple segments labeling
Each segment overwrites bounds of previous one.
They should be added instead

### PR DESCRIPTION
**Pull request by VargBurz**
Closes #658 
## Problem: 
When you label multiple segments in anomaly detector, every new segment delete previous seasonality from smoothed dataset.
For example for seasonity 1 day:
label first segment in 16:00 :
image
label second segment in 21:00 :
image
label both:
image

## Changes:
At first, smoothed borders are determined before the loop, and then seasonality is added to it.

For example for seasonity 1 day:
label first segment in 16:00 :
image
label second segment in 21:00 : 
image
label both:
image

### CODE REVIEW

File: analytics/analytics/detectors/anomaly_detector.py [hunk 1]
```diff
@@ -87,18 +87,16 @@ def detect(self, dataframe: pd.DataFrame, cache: Optional[ModelCache]) -> Detect
                 seasonality_index = seasonality // time_step
                 #TODO: upper and lower bounds for segment_data
                 segment_data = utils.exponential_smoothing(pd.Series(segment['data']), BASIC_ALPHA)
-                upper_seasonality_curve = self.add_season_to_data(
-                    smoothed_data, segment_data, seasonality_offset, seasonality_index, True
+                upper_bound = self.add_season_to_data(
+                    upper_bound, segment_data, seasonality_offset, seasonality_index, True
```
Comment (rozetko) [inline analytics/analytics/detectors/anomaly_detector.py]: `upper_bound` is not defined at 1st iteration

File: analytics/analytics/detectors/anomaly_detector.py [hunk 2]
```diff
@@ -87,18 +87,16 @@ def detect(self, dataframe: pd.DataFrame, cache: Optional[ModelCache]) -> Detect
...
                 )
-                lower_seasonality_curve = self.add_season_to_data(
-                    smoothed_data, segment_data, seasonality_offset, seasonality_index, False
+                lower_bound = self.add_season_to_data(
+                    lower_bound, segment_data, seasonality_offset, seasonality_index, False
```
Comment (rozetko) [inline analytics/analytics/detectors/anomaly_detector.py]: same thing for `lower_bound`

File: analytics/analytics/detectors/anomaly_detector.py (hunk 1 above)
Comment (VargBurz) [reply analytics/analytics/detectors/anomaly_detector.py]: It is already defined.
Before segments loop

File: analytics/analytics/detectors/anomaly_detector.py (hunk 2 above)
Comment (VargBurz) [reply analytics/analytics/detectors/anomaly_detector.py]: same thing for lower_bound

**Pull request merged**